docs/faq.md

```diff
@@ -0,0 +1,89 @@
+---
+Title: Frequently Asked Questions
+---
+
+## General Questions
+
+### Q: What is Headlamp and who is it for?
+
+**A:** Headlamp is graphical interface specifically tailored for Kubernetes users to simplify the management of Kubernetes clusters.

```

Comment:
"is *a* graphical *user* interface".
Maybe
`**A:** Headlamp is a graphical user interface specifically tailored for simplifying the management of Kubernetes clusters.`

Reply:
done!